pages/common/pastel.md

```diff
@@ -0,0 +1,24 @@
+# pastel
+
+> Generate, analyze, convert and manipulate colors.
+> More information: <https://github.com/sharkdp/pastel>.
+
+- Convert colors from one format to another:
```

Comment:
It would be helpful to mention the source and target format here in the description. It is not very clear from the example.

Reply:
Thanks for the input. There are a lot of formats to chose from (rgba, hsl…) Which sentence do you suggest do describe this command?

Reply:
Just mention what the example command is doing. I believe it is converting rgba to hsl ?

Reply:
Ok, I've precised what this command does.